prediction_market_agent/agents/microchain_agent/deploy.py

```diff
@@ -71,106 +108,99 @@ def run(
         self.run_general_agent(market_type=market_type)
 
     @observe()
-    def run_general_agent(
-        self,
-        market_type: MarketType,
-    ) -> None:
-        self.langfuse_update_current_trace(tags=[GENERAL_AGENT_TAG, self.identifier])
-
-        long_term_memory = LongTermMemoryTableHandler.from_agent_identifier(
-            self.identifier
-        )
-        prompt_handler = PromptTableHandler.from_agent_identifier(self.identifier)
-        unformatted_system_prompt = get_unformatted_system_prompt(
-            unformatted_prompt=self.get_initial_system_prompt(),
-            prompt_table_handler=prompt_handler,
-        )
+    def run_general_agent(self, market_type: MarketType) -> None:
+        if market_type != MarketType.OMEN:
```

Comment:
PMAT needs a little refactor to pass `MarketType` into `load` method, hotfixing it like this for now here.